crates/networking/p2p/sync.rs

```diff
@@ -292,63 +318,82 @@ impl SyncManager {
                 // Next sync will be full-sync
                 self.sync_mode = SyncMode::Full;
             }
-            SyncMode::Full => {
-                // full-sync: Fetch all block bodies and execute them sequentially to build the state
-                download_and_run_blocks(
-                    all_block_hashes,
-                    self.peers.clone(),
-                    store.clone(),
-                    &mut self.invalid_ancestors,
-                    &self.blockchain,
-                )
-                .await?
-            }
+            // Full sync stores and executes blocks as it ask for the headers
```

Comment:
```suggestion
            // Full sync stores and executes blocks as it asks for the headers
```

Does Geth also start downloading bodies before it finishes downloading headers?

Reply:
Because by only downloading block headers and verifying them (that the chain is correct) you can have an "optimistic head". This is probably out of scope of this PR though.

Reply:
Solved typo in [017c7cc](https://github.com/lambdaclass/ethrex/pull/2061/commits/017c7cc44dad37b41cc273a05963ba31c1028845).